core/arch/instr.h

```diff
@@ -325,6 +325,26 @@ typedef enum _dr_encoding_hint_type_t {
 #endif
 } dr_encoding_hint_type_t;
 
+/** AVX-512 tuple type attributes as specified in Intel's tables. */
+typedef enum dr_tuple_type_t {
```

Comment:
We always use a leading underscore on the two-word type name for structs and enums.

Reply:
Fixed.